annotations/application-crd-annotations/src/main/java/io/ap4k/application/annotation/Info.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2018 The original authors.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+**/
+
+package io.ap4k.application.annotation;
+
+public @interface Info {
```

Comment:
FYI. I have opened a ticket on k8s Application CRD project in order to have the possibility to pass k8s objects here such as List of Env, .... to let by example the operator to configure the Deployment/Deploymentconfig. The existing field `valueFrom` will select either a secret, configMap, ingress, ... resource